module_3/assignment1_2.js

```diff
@@ -0,0 +1,14 @@
+let age = 25;
+
+switch (true){
+  case (age <= 12):
+    console.log("child");
+    break;
+  case (18 <= age >= 13):
```

Comment:
Please check this will not work condition is wrong.

Reply:
Done